sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeLinkedEntityAsyncClient.java

```diff
@@ -101,7 +102,7 @@
      * @return A mono {@link Response} that contains {@link RecognizeLinkedEntitiesResultCollection}.
      */
     Mono<Response<RecognizeLinkedEntitiesResultCollection>> recognizeLinkedEntitiesBatch(
-        Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options) {
+        Iterable<TextDocumentInput> documents, RecognizeLinkedEntitiesOptions options) {
```

Comment:
This is a breaking change.

Reply:
This is an internally method. I add a new API to support RecognizeLinkedEntitiesOptions.
https://github.com/Azure/azure-sdk-for-java/pull/18984/files#diff-f772d27ac3cc6b57b1654f337f785208250b0a9191549584195354d3aa3d9ed7R735